cmd/ftl/cmd_schema.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 type schemaCmd struct {
-	Get      getSchemaCmd      `default:"" cmd:"" help:"Retrieve the cluster FTL schema."`
```

Comment:
We want to preserve this.

Reply:
Changed it up so we can preserve it. See the PR description for the new interface.
Or is there a way to get the non-named arg within `ftl schema get` with kong while maintaining the default cmd for `ftl schema`? I figured you'd be the expert on that ;)